source/franka_matlab/getting_started.rst

```diff
@@ -46,48 +52,43 @@ Execution
 .. hint::
 
     The current workflow presented is utilizing the "Run on Custom Hardware" Simulink App, present in Matlab versions
-    :math:`\geq` R2020a. In case of Matlab 2019a you can build the model normally by using the "Build Model" button.
-    You then need to run the executable from a terminal as described below.
+    :math:`\geq` R2020a. In case of Matlab 2019a you can build the model normaly by using the "Build Model" button.
+    You then need to run the executable from a terminal as described bellow.
 
 Let's start by selecting the `Run on custom hardware` App from the Apps pane in Simulink.
 
-.. figure:: _static/simulink_apps_pane.png
+.. figure:: _static/cartesian_impedance_control_apps.png
     :align: center
     :figclass: align-center
 
-    Run on custom hardware Simulink App.
+    "Run on custom hardware" Simulink App.
+
+.. note::
+
+    The Franka Simulink Library has been tested with the Generic Real-Time Target (grt.tlc) as well as with the Embedded Coder Target (ert.tlc) in the context of the NVIDIA's Jetson Nano Hardware support package for Simulink.
 
 .. important::
 
     Before executing make sure that the brakes of the robot are disengaged and that the robot is in execution mode!
 
-You can then select from the Hardware tab either `Monitor & Tune` in case monitoring through the external mode is
+You can then select from the Hardware tab either `Monitor & Tune` in case monitoring through the external mode is 
 desired or `Build, Deploy & Start` for just executing the application without monitoring.
 
-.. figure:: _static/simulink_hardware_pane.png
+.. figure:: _static/cartesian_impedance_control_hardware.png
     :align: center
     :figclass: align-center
 
     Hardware Simulink App.
 
-For running the generated executable manually from a terminal make sure that you've first exported the full libfranka
-build path, in case of Linux:
-
-.. code-block:: shell
-
-    >> $ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<libfranka build folder full path>
-
-or in case of Windows, that you've included the full path of libfranka build directory in the PATH environment variable.
-
 .. hint::
 
-    As a reminder, in case of a robot with system version 4.2.0 or greater, the FCI control mode needs to be explicitly enabled through Desk --> Sidebar menu --> Activate FCI.
+    As a reminder, in case of a robot with system version 4.2.0 or greater, the FCI control mode needs to be explicitly enabled through Desk --> Sidebar menu --> Activate FCI. 
 
 .. caution::
 
     The robot will move! Make sure that you are monitoring the situation, ready to take action if necessary!
 
-You can then run the executable, which is located in the current working space.
+Alternatively you can run the executable, which is located in the current working space, manually.
 
 In case of Linux:
```

Comment:
In case of running the executable after pressing `Build, Deploy & Start`, the robot ip should be added after ./<simulink_model_name>.
In case of running the executable after using ` Monitor & Tune`, the duration and port should be added after ./<simulink_model_name> in case we want to run it again manually